src/win/process.c

```diff
@@ -372,7 +374,7 @@ static WCHAR* search_path(const WCHAR *file,
         file, file_name_start - file,
         file_name_start, file_len - (file_name_start - file),
         cwd, cwd_len,
-        name_has_ext);
+        1);
```

Comment:
Please update the above `don't use path` comment to explain that we try the exact name given first, and then check for standard extensions.